pino.js

```diff
@@ -272,6 +272,17 @@ Object.defineProperty(pinoPrototype, 'isLevelEnabled', {
   value: isLevelEnabled
 })
 
+function getPrettyStream (opts) {
+  var prettyFactory
+  var prettyModule = opts.prettifier || 'pino-pretty'
+  try {
+    prettyFactory = require(prettyModule)
```

Comment:
I would not require a string, but rather support a function (the factory) or true. If true, we can then require.

Reply:
That makes sense.